src/WebJobs.Extensions.DurableTask/RetryOptions.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using DurableTaskCore = DurableTask.Core;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    ///     Contains retry policies that can be passed as parameters to various operations
+    /// </summary>
+    public class RetryOptions
+    {
+        private DurableTaskCore.RetryOptions retryOptions;
+
+        /// <summary>
+        /// Creates a new instance RetryOptions with the supplied first retry and max attempts
+        /// </summary>
+        /// <param name="firstRetryInterval">Timespan to wait for the first retry</param>
+        /// <param name="maxNumberOfAttempts">Max number of attempts to retry</param>
+        /// <exception cref="ArgumentException"></exception>
+        public RetryOptions(TimeSpan firstRetryInterval, int maxNumberOfAttempts)
+        {
+            retryOptions = new DurableTaskCore.RetryOptions(firstRetryInterval, maxNumberOfAttempts);
+        }
+
+        /// <summary>
+        /// Gets or sets the first retry interval
+        /// </summary>
+        public TimeSpan FirstRetryInterval
```

Comment:
I realize this may be a bit of a drag, but the automatic API doc generation tools require that we include <value> tags for each public property we expose. Can you add those to all the properties in this class? Look at the property documentation for other classes, like `DurableOrchestrationContext` for an example of how I've been doing this.